src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        int index = new Random().nextInt(Color.values().length);
+        return Color.values()[index].toString().toLowerCase();
```

Comment:
The `getRandomColor` method returns a lowercase string representation of the color. If the `Ball` class or other parts of the application expect a `Color` type, consider returning `Color.values()[index]` directly instead of converting it to a string.